apps/sms/test/marionette/lib/views/conversation/message_accessors.js

```diff
@@ -52,6 +54,16 @@ MessageAccessor.prototype = {
     }
   },
 
+  selectMessage: function(messageNode) {
+    if (this.parse(messageNode).isSelected) {
+      throw new Error('Message has been selected!');
+    }
+    //messageNode.tap();
+    //messageNode.findElement('li[class=message]').tap();
+    //messageNode.findElement('.pack-checkbox').click();
+    messageNode.findElement('.pack-checkbox').tap();
```

Comment:
In the mean time - what happens when you tap on the ```messageNode```? Doesn't it work as expected?